plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ConditionalRouterMediatorPropertiesEditionPartForm.java

```diff
@@ -137,12 +136,11 @@ public Composite createFigure(final Composite parent, final FormToolkit widgetFa
 	public void createControls(final FormToolkit widgetFactory, Composite view) {
 		CompositionSequence conditionalRouterMediatorStep = new BindingCompositionSequence(propertiesEditionComponent);
 		CompositionStep propertiesStep = conditionalRouterMediatorStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.class);
-		propertiesStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.description);
 		propertiesStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.commentsList);
 		propertiesStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.reverse);
 		propertiesStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.continueAfterRoute);
 		propertiesStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.conditionalRouteBranches);
-		
+        propertiesStep.addStep(EsbViewsRepository.ConditionalRouterMediator.Properties.description);
```

Comment:
Fix formatting.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/806